validation.md

```diff
@@ -2241,6 +2244,211 @@ Occasionally, you may want to attach additional validation rules to your default
+
+```php
+// Basic RFC compliance...
+Rule::email()->rfcCompliant();
+
+// Strict RFC compliance...
+Rule::email()->rfcCompliant(strict: true);
+
+// Check for valid MX records...
+Rule::email()->validateMxRecord();
+
+// Prevent spoofing...
+Rule::email()->preventSpoofing();
+```
+Of course, you may chain all the methods in the examples above:
+
+```php
+Rule::email()
+    ->rfcCompliant(strict: true)
+    ->validateMxRecord()
+    ->preventSpoofing();
+```
+
+> [!WARNING]  
+> The `validateMxRecord()` and `preventSpoofing()` validators require the PHP `intl` extension.
+
+<a name="defining-default-email-rules"></a>
+#### Defining Default Email Rules
+You may find it convenient to specify the default validation rules for emails in a single location of your application. You can easily accomplish this using the Email::defaults method, which accepts a closure. The closure given to the defaults method should return the default configuration of the `Email` rule. Typically, the `defaults` rule should be called within the boot method of one of your application's service providers:
+
+```php
+use Illuminate\Validation\Rules\Email;
+
+/**
+ * Bootstrap any application services.
+ */
+public function boot(): void
+{
+    Email::defaults(function () {
+        $rule = (new Email())->rfcCompliant(strict: true)->preventSpoofing(),
+
+        return $this->app->isProduction()
+                    ? $rule->validateMxRecord()
+                    : $rule;
+    });
+}
+```
+Then, when you would like to apply the default rules to a particular email undergoing validation, you may invoke the defaults method with no arguments:
+
+```php
+'email' => ['required', Email::defaults()],
+```
+Occasionally, you may want to attach additional validation rules to your default email validation rules. You may use the rules method to accomplish this:
+
+```php
+Email::defaults(function () {
+    return (new Email())->rfcCompliant(strict: true)
+      ->preventSpoofing()
+      ->rules(['ends_with:@example.com']);
+});
+```
+
+<a name="email-validation-use-cases-examples"></a>
+### Use Cases & Examples
+
+Each method on the `Email` rule addresses a different aspect of email validation.
+
+<a name="rfc-compliant-email-validation"></a>
+#### `rfcCompliant(strict: bool)`
+
+**What it does**
+- Validates the email according to [RFC 5322](https://datatracker.ietf.org/doc/html/rfc5322).
+- Passing `true` enforces stricter RFC checks (e.g., rejects trailing dots or multiple consecutive dots).
+
+**When to use it**
+- Use `rfcCompliant()` for typical RFC validation while allowing some uncommon formats.
+- Use `rfcCompliant(strict: true)` to reject unusual but technically valid addresses.
+
+**Email addresses that pass on `rfcCompliant()` but fail on `rfcCompliant(strict: true)`**
+  - `test@example` (No TLD)
+  - `"has space"@example.com` (local part ends with a dot)
```

Comment:
```suggestion
  - `test.@example.com` (local part ends with a dot)
  - `"has space"@example.com` (quoted local part with space)
```

Reply:
Thanks, fixed the examples/text in that area